pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -240,19 +244,16 @@ func (s *Service) getCachedUserPermissions(ctx context.Context, user identity.Re
 		return nil, err
 	}
 
-	permissions := make([]accesscontrol.Permission, 0, len(basicRolesPermissions)+len(teamsPermissions)+len(userPermissions))
-	permissions = append(permissions, basicRolesPermissions...)
-	permissions = append(permissions, teamsPermissions...)
 	permissions = append(permissions, userPermissions...)
+
```

Comment:
It'd be interesting to record the # of permissions for posterity, how about we add it as a trace attribute:

```suggestion
span.SetAttributes(attribute.Int("num_permissions", len(permissions)))

```

Reply:
need to import `"go.opentelemetry.io/otel/attribute"` too 😅